src/modules/graphics/graphics.c

```diff
@@ -2045,16 +2067,17 @@ float lovrFontGetWidth(Font* font, ColoredString* strings, uint32_t count) {
 }
 
 void lovrFontGetLines(Font* font, ColoredString* strings, uint32_t count, float wrap, void (*callback)(void* context, const char* string, size_t length), void* context) {
-  size_t totalLength = 0;
+  uint32_t totalLength = 0;
```

Comment:
I think it would be cleaner to change the cursor to `size_t` instead of making this a `uint32_t`.